todo-app/src/todo/controller.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    Body,
+    Controller,
+    Delete,
+    Get,
+    Param,
+    ParseIntPipe,
+    Post,
+    Put,
+  } from '@nestjs/common';
+  import { TodoService } from './service';
+  import {
+    CreateTodoRequestDto,
+    GetTodoDateRequestDto,
+  } from './dto/request.dtos';
+  import { TodoResponseDto } from './dto/response.dtos';
+  
+  @Controller('todo')
```

Comment:
- this module name is preferably plural
- if this was called task module, then we would have `@Controller('tasks')`
- even if you went with the word todo for the module name, you should have used `@Controller('todos')`